src/pages/workspaces/workspace/ShareWorkspaceModal/ShareWorkspaceModal.ts

```diff
@@ -165,7 +165,7 @@ const ShareWorkspaceModal: React.FC<ShareWorkspaceModalProps> = (props: ShareWor
                 onChange: setSearchValue,
                 suggestions: cond(
                   [searchValueValid && !_.includes(searchValue, aclEmails), () => [searchValue]],
-                  [remainingSuggestions.length, () => remainingSuggestions],
+                  [remainingSuggestions.length > 0, () => remainingSuggestions as unknown as string[]],
```

Comment:
Not sure why `remainingSuggestions` is typed as some Lodash type instead of a `string[]`.

Reply:
Ah, TS couldn't figure out what `remainingSuggestions` is because `suggestions` is typed `any`. Typing `suggestions` as `string[]` made `remainingSuggestions` also typed `string[]`.

Resolved in https://github.com/DataBiosphere/terra-ui/pull/4161/commits/0ae27b4fe08d1af0a09e1392c5af8f32070ac22f.